pootle/apps/accounts/models.py

```diff
@@ -241,6 +242,16 @@ def delete(self, *args, **kwargs):
         if self.is_meta:
             raise ProtectedError('Cannot remove meta user instances', None)
 
+        purge = kwargs.pop("purge", False)
+
+        if purge:
+            purge_user(self)
+        else:
+            try:
+                merge_user(self, User.objects.get_nobody_user())
+            except User.DoesNotExist:
```

Comment:
in tests its not always present - which is why i had been concerned.

Ill add the fixture to the tests that require it - but it does mean that tests using User.delete might fail if they are missing the fixture